src/internal/observable/combineLatest.ts

```diff
@@ -545,7 +545,7 @@ class CombineLatestSubscriber<T> extends Subscriber<T> {
 
 export function combineLatestInit(
   observables: ObservableInput<any>[],
-  scheduler: SchedulerLike | undefined = undefined,
+  scheduler?: SchedulerLike | undefined,
```

Comment:
nitpick: If `scheduler` is optional, the `undefined` is implied and doesn't have to be included in the union.